python/gps/utility/visualization.py

```diff
@@ -0,0 +1,141 @@
+from gps.proto.gps_pb2 import END_EFFECTOR_POINTS, RGB_IMAGE
+import numpy as np
+import matplotlib.pyplot as plt
+
+TRIALS = 20
+THRESH = {'reacher': 0.05, 'pointmass': 0.1}
+
+def compare_samples_curve(gps, N, agent_config, weight_varying=False, experiment='reacher'):
+    """
+    Compare samples between IOC and demo policies and visualize them in a plot.
+    Args:
+        gps: GPS object.
+        N: number of samples taken from the policy for comparison
+        config: Configuration of the agent to sample.
+        weight_varying: whether the experiment is weight-varying or not.
+        experiment: whether the experiment is reacher or pointmass.
+    """
+    pol_iter = gps._hyperparams['algorithm']['iterations'] - 1
+    algorithm_ioc = gps.data_logger.unpickle(gps._data_files_dir + 'algorithm_itr_%02d' % pol_iter + '.pkl')
+    algorithm_sup = gps.data_logger.unpickle(gps._hyperparams['common']['supervised_exp_dir'] + 'data_files/algorithm_itr_%02d' % pol_iter + '.pkl')
+    algorithm_demo = gps.data_logger.unpickle(gps._hyperparams['common']['demo_exp_dir'] + 'data_files/algorithm_itr_09.pkl') # Assuming not using 4 policies
```

Comment:
Isn't this value in the hyperparams directory as demo_controller_file, or something similar?